static-analysis/StaticAnalysisChapter.md

```diff
@@ -366,13 +365,13 @@ julia> code_typed(lloop, (Int,))[1].args[3]
     end::Nothing)
 ~~~
 
-You'll notice there's no `for` or `while` loop in the body. As the compiler transforms the code from what we wrote to the binary instructions the CPU understands, features that useful to humans but that are not understood by the CPU (like loops) are removed. The loop has been rewritten as `label`s and `goto`s. The `goto` has a number in it; each `label` also has a number. The `goto` jumps to the the `label` with the same number.
+You'll notice there's no for or while loop in the body. As the compiler transforms the code from what we wrote to the binary instructions the CPU understands, features that are useful to humans but that are not understood by the CPU (like loops) are removed. The loop has been rewritten as `label` and `goto` commands. The `goto` has a number in it; each `label` also has a number. The `goto` jumps to the the `label` with the same number.
 
-### Detecting and Extracting Loops
+#### Detecting and Extracting Loops
 
-We're going to find loops by looking for `goto`s that jump backwards.
+We're going to find loops by looking for `goto` commands that jump backwards.
```

Comment:
I don't think the `goto`s are commands. I guess "expressions" would be more accurate?